sdk/monitor/azquery/models.go

```diff
@@ -108,7 +110,11 @@ type LogsClientQueryBatchOptions struct {
 
 // LogsClientQueryWorkspaceOptions contains the optional parameters for the LogsClient.QueryWorkspace method.
 type LogsClientQueryWorkspaceOptions struct {
-	// Optional. The prefer header to set server timeout, query statistics and visualization information.
+	// Optional. The prefer header to set server timeout, query statistics and visualization information. For server timeout,
+	// set the prefer string equal to "wait=NumOfMinutes" where NumOfMinutes is a
```

Comment:
```suggestion
	// set the prefer string equal to "wait=NumOfMinutes", where NumOfMinutes is an
```

Reply:
fixed in commit [485540e](https://github.com/Azure/azure-sdk-for-go/pull/19771/commits/485540ef7be80478d1ee36abdb80a71c3e86a66e)